meta-resin-common/recipes-containers/docker/docker/docker.conf.systemd

```diff
@@ -0,0 +1,3 @@
+[Service]
+ExecStart=
+ExecStart=/usr/bin/docker daemon --log-driver=journald -s btrfs -H fd:// -H tcp://0.0.0.0:2375/
```

Comment:
@lekkas Can give much more info about this. But generally, users will be able to control the docker daemon from another host on the network.